### DISCUSSION TITLE
SRT connection closing

### DISCUSSION THREAD

**Discussion by jhernandez562** [bluenviron/mediamtx#2587] [Questions]
### Question

Hello all,

I am trying to establish a SRT feed, I connect and Mediamtx console displays the following: 
[SRT] [conn *ip address:port*] opened 
[SRT] [conn *ip address:port*] is publishing to path 'stream/ch2', 1 track (MPEG-4 Audio)

and then a few seconds later I get:

[SRT] [conn *ip address:port*] closed: astits: building new data failed: astits: parsing PES data failed: astits: fetching next bytes failed: astikit: slice length is 2574, offset 4152 is invalid

Why is this happening? I only get this problem when trying to publish stream on phone via 5G. I  have tried it with RTMP with success. I am using version 1.2.

**Comment by aler9** [comment]
Hello, what software and what device are you using to publish to the server with SRT?

[a comment by an automated account is not shown]
